src/app-layout/__tests__/multi-layout.test.tsx

```diff
@@ -89,7 +108,9 @@ describeEachAppLayout({ themes: ['refresh-toolbar'], sizes: ['desktop'] }, () =>
     expect(createWrapper().findAllByClassName(testUtilStyles.tools)).toHaveLength(1);
 
     firstLayout.findToolsToggle().click();
-    expect(secondLayout.findOpenToolsPanel()).toBeTruthy();
+    expect(secondLayout.findNavigation()).toBeFalsy();
+    expect(secondLayout.findNavigationToggle()).toBeFalsy();
+    expect(secondLayout.findOpenToolsPanel).toBeTruthy();
```

Comment:
Missed to call the method

```suggestion
    expect(secondLayout.findOpenToolsPanel()).toBeTruthy();
```